readme.md

```diff
@@ -1,44 +1,65 @@
-## Raft API
+## [RTAPI] Raft API
 
-Raft is a online raffle site.
+RTAPI is the laravel backend for the raft platform. It's REST api is consumed by the [RTUI](https://github.com/badleyy/rtui) and other cosumers.
 
-## Official Documentation
+## Installation
 
-### Installation
+The API is a [laravel](https://laravel.com) php application built only to provide an http api interface to the RAFT platform.
 
-#####Clone the repository
-  "git clone https://github.com/badleyy/rtapi.git"
+### Clone the repository
 
-#####Install dependencies 
-  "composer update"* at the root directory
+```
+$ git clone https://github.com/badleyy/rtapi.git
+```
 
-#####Generate a site key and update the .env file
-  "php artisan key:generate"</br>
-  ./.env file - APP_KEY={generate key}
+### Install dependencies 
 
-#####Update the laravel service providers and aliases to include jwt authenciation
-  ./config/app.php - Add "Tymon\JWTAuth\Providers\JWTAuthServiceProvider::class" in the providers array</br>
-  ./config/app.php - Add to the aliases array</br>
-  'JWTAuth' => Tymon\JWTAuth\Facades\JWTAuth::class,</br>
-  'JWTFactory' => Tymon\JWTAuth\Facades\JWTFactory::class
```

Comment:
Correct, you should only have to just install the correct package using composer. The configurations are already check into the repository.